errors/errors.go

```diff
@@ -3,6 +3,8 @@ package errors
 import (
 	"fmt"
 
+	"strings"
```

Comment:
I'm giving up on GoLand's `goimport` functionality, it's broken and needs to use the `goimports` utility written in Go.  Something to add to your feature request along with official `gorename` support.  :~]